Lib/unittest/case.py

```diff
@@ -362,18 +362,20 @@ def __iter__(self):
 class TestCase(object):
     """A class whose instances are single test cases.
 
-    By default, the test code itself should be placed in a method named
-    'runTest'.
+    Test authors should subclass this TestCase class for their own tests.
+    Testing code may be placed in subclass method named 'runTest';
+    alternatively, invoke unittest.main() from the subclass's "__name__ guard"
+    section to run all methods defined in the test class. To achieve more
+    precise subclass test invocation see below (viz., constructor kwarg).
```

Comment:
The ``runTest`` form is far rarer, should we cover it second?

Reply:
I rewrote the docs according to your suggestion.